raiden-ts/src/helpers.ts

```diff
@@ -491,10 +490,8 @@ function validateDump(
  * @param deps.network - current network
  * @param deps.contractsInfo - current contracts
  * @param deps.log - Logger instance
- * @param storage - either [[Storage]] or [[RaidenState]] or
- * { storage: [[Storage]]; state?: [[RaidenState]] }
+ * @param storage - diverse storage related parameters to load from and save to
  * @param storage.state - Uploaded state: replaces database state; must be newer than database
- * @param storage.storage - Like above, but load from localStorage
  * @param storage.adapter - RxDB adapter: usually 'indexeddb' or [leveldown] module object
```

Comment:
Please, can you also fix this description? It's not RxDB, but PouchDB adapter. Same for `prefix` below

Reply:
Oh. This is an ancient mistake. :laughing:

Reply:
Should it be described as for the `getState` function? There it got adopted.

Reply:
It's the `getState` which is wrong; yes, you should be able to copy it from `Raiden.create`. Thanks

Reply:
Sorry, twisted it.

Reply:
:heavy_check_mark: